src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -301,6 +308,16 @@ def __call__(cls, *_args: Any, **_kwargs: Any) -> NoReturn:
         )
 
 
+class Temperature(
```

Comment:
Could you split adding the `Temperature` quantity into a separate commit? This way it is easier to focus on a particular change during the review.